plugins/out_kinesis_firehose/firehose.c

```diff
@@ -508,6 +520,13 @@ static struct flb_config_map config_map[] = {
      "AWS Profile name. AWS Profiles can be configured with AWS CLI and are usually stored in "
      "$HOME/.aws/ directory."
     },
+
+    {
+     FLB_CONFIG_MAP_BOOL, "simple_aggregation", "false",
```

Comment:
This is just my curiosity but is there any reasons to use `simple_` prefix in this config parameter?
This is because if this aggregation mode is only existing to aggregate records, we don't need to use `simple_` prefix here. We can just use `aggregation` mode here.

Reply:
Part of an effort to port over the functionality from the golang plugins - https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit where it was originally `simple_aggregation`. I think there was a more complex approach to take using the Kinesis Producer Library (KPL) but that may have required additional dependencies to bring it over to fluent-bit